### PR TITLE
Add ability to accept footer labels

### PR DESCRIPTION
**Pull request by Aldizh**
Please refer to this PR and merge both at the same time so that we have no issues:
https://github.com/copartit/ops_portal/pull/2645

### CODE REVIEW

File: src/Footer.js [hunk 1]
```diff
@@ -145,10 +145,10 @@ class RowSelectTableFooter extends Component {
                 {pageSizeOptions.map((pageSizeOption, i) => <MenuItem value={i} key={i} primaryText={pageSizeOption} />)}
               </Menu>
             </Popover>
-            {`  Results${resultCount ? ` of ${resultCount}` : ''}`}
+            {`  ${footerLabels.results}${resultCount ? ` ${footerLabels.of} ${resultCount}` : ''}`}
```
Comment (Shobhit1) [inline src/Footer.js]: Hey @Aldizh, can you please add a `footerLabels` defaultProps here?

Comment (Aldizh) [inline src/Footer.js]: done

Comment (anilreddykatta) [inline src/Footer.js]: Can we declare `resultCount` at the top of the render function and keep this templating clean with only one variable
```
render() {
   const resultCount = this.props.resultCount ? ` ${footerLabels.of} ${resultCount}` : ''
    return (
      // 148
      {`${footerLabels}${resultCount}`}
    )
}
```

File: src/Footer.js [hunk 2]
```diff
@@ -185,12 +185,23 @@ class RowSelectTableFooter extends Component {
...
+  }
+}
+
 RowSelectTableFooter.propTypes = {
   maxPage: PropTypes.number,
   currentPage: PropTypes.number,
   setPage: PropTypes.func,
   resultsPerPage: PropTypes.number,
   pageSizeOptions: PropTypes.arrayOf(PropTypes.number).isRequired,
+  footerLabels: PropTypes.objectOf(PropTypes.any),
```
Comment (anilreddykatta) [inline src/Footer.js]: Mention the exact shape?
```
footerLabels: PropTypes.shape({
    showing: PropTypes.string,
    results: PropTypes.string,
    go: PropTypes.string,
    of: PropTypes.string,
    goToPage: PropTypes.string,
})
```

**Comment by Aldizh** [comment]
Added the two suggestions.
Can you merge this now as it has been opened for quite a while...

**Pull request merged**